src/main/java/com/pinterest/secor/io/KeyValue.java

```diff
@@ -25,16 +25,29 @@
  */
 public class KeyValue {
 	
-	private final long mKey;
+	private final long mOffset;
+	private final byte[] mKey;
```

Comment:
There is only going to be getKafkaKey() and getOffset() methods in the new class.  This would force all the client to rename the invocation of getKey() to getOffset().  I was worried if we reusing the getKey() with different meanings, some of the client might forget to rename 'getKey()' to 'getOffset'.